projects/plugins/boost/app/assets/src/js/utils/connection.ts

```diff
@@ -11,5 +11,8 @@ export async function onConnectionComplete(): Promise< void > {
 		await requestCloudCss();
 	}
 
+	// Set a flag to indicate that the user has to get started with the plugin.
+	getStarted.setValue( true );
```

Comment:
🤔 I gather this is setting a local flag to force the user over to the getting started screen? I don't think this will take the wp_options value into account.

Would it be simpler / make more sense to always send the user to the "getting started" screen after connection unless they've already bought Boost premium? That way, we don't have to keep track of a back-end flag, and we have a really simple startup flow.

Reply:
I think the difference between the two approaches is negligible, and will only impact edge-case users. I'm fine to go with your model. :)